### ISSUE TITLE
Update mol2tosdf.py to not assign new charges

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/SolvationToolkit#50]
@camizanette - can you update the `mol2tosdf.py` functionality (writeSDF, particularly) to NOT modify partial charges? Our main use case is to apply this to `.mol2` files which already have partial charges and switch them to `.sdf` files with identical partial charges. Your code has:

```
oequacpac.OEAssignPartialCharges( molToCharge, oequacpac.OECharges_AM1BCCSym )
```

While this assigns charges in an identical way to what we currently use in the `.mol2` files, conceivably we might later switch to a different charging method for generating the `.mol2` files and we want to ensure we have the same charges in the `.sdf` files. So it would be better just to use the charges straight from the `.mol2` file without any reassignment.

Also, it would be great if you could make the doc strings for the functions a tiny bit more complete - i.e. specifying what the input and output is. Ideally you want the doc string to be enough to understand how to use the function without reading any code. For example, for writeSDF, it's not totally clear what mol_name does (without reading the code) and the doc string doesn't actually say that this operates on an existing .mol2 file, creates a new .sdf file (or overwrites an existing one) containing the same molecule, and returns nothing. Also, I'm unclear what happens if, for example, the bond order is listed as 'ar' (isn't it sometimes?).

**Comment by camizanette** [comment]
The 'ar' bond order is translate to 2 or 1. So, for example,benzene will be something like 2,1,2,1,2,1 for representing the Carbon-Carbon bond orders.

**Comment by davidlmobley** [comment]
Do you translate it, or is that something done by OpenEye tools?

On Tue, Apr 26, 2016 at 5:27 PM, Camila <notifications@github.com> wrote:

> The 'ar' bond order is translate to 2 or 1. So, for example,benzene will
> be something like 2,1,2,1,2,1 for representing the Carbon-Carbon bond
> orders.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MobleyLab/SolvationToolkit/issues/50#issuecomment-214928459>
>



-- 
David Mobley
dmobley@gmail.com
949-385-2436

**Comment by camizanette** [comment]
It's done by OpenEye tools (OEIFlavor).

**Comment by davidlmobley** [comment]
OK, that's fine then. Can you make the other change described here?

**Comment by davidlmobley** [comment]
Resolved by #54
